projects/shared-modules/mention/components/suggestion-box/suggestion-box.component.scss

```diff
@@ -0,0 +1,74 @@
+@import '/projects/commudle-admin/src/assets/styles/variables';
+
+.suggestions-box{
+  width: 300px;
+}
+
+.mention-menu {
+  position: absolute;
+  left: 0;
+  z-index: 1000;
+  float: left;
+  min-width: 11em;
+  border: 2px solid rgba(0, 0, 0, 0.15);
+  border-radius: 0.25em;
+}
+
+// .mention-item {
+//   display: block;
+//   padding: 0.2em 1.5em;
+//   line-height: 1.5em;
+//   clear: both;
+//   font-weight: 400;
+//   color: #212529;
+//   text-align: inherit;
+//   white-space: nowrap;
+//   background-color: transparent;
+//   border: 0;
+// }
+
+.scrollable-menu {
+  display: block;
+  height: auto;
+  max-height: 292px; /* fits 11 items with 14px font-size */
+  overflow: auto;
+
+  // Custom scrollbar
+  &::-webkit-scrollbar {
+    width: 0;
+  }
+}
+
+.mention-dropdown {
+  bottom: 100%;
+  top: auto;
+  margin-bottom: 2px;
+  background: white;
+}
+
+.selected{
+  background-color: cornflowerblue;
```

Comment:
Let's use variables for colors, if the color you need doesn't have a variable, then create one